quickstart/201-web-app-postgres-keyvault/mysql.tf

```diff
@@ -1,29 +1,39 @@
 resource "azurerm_mysql_server" "webAppBackend" {
-  name                              = "${var.siteName}pgserver"
-  location                          = "${azurerm_resource_group.webAppMySqlRg.location}"
-  resource_group_name               = "${azurerm_resource_group.webAppMySqlRg.name}"
-  tags                              = "${azurerm_resource_group.webAppMySqlRg.tags}"
+  name                = "${replace(random_pet.prefix.id, "-", "")}pgserver"
+  location            = azurerm_resource_group.webAppMySqlRg.location
+  resource_group_name = azurerm_resource_group.webAppMySqlRg.name
+  tags                = azurerm_resource_group.webAppMySqlRg.tags
 
-  administrator_login               = "${var.administratorLogin}"
-  administrator_login_password      = "${var.administratorLoginPassword}"
-  version                           = "${var.mysqlVersion}"
-  ssl_enforcement                   = "Disabled"
-  sku {

```

Comment:
Delete this sku block according to https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mysql_server#sku_name